client/layout/index.jsx

```diff
@@ -143,6 +143,7 @@ Layout = React.createClass( {
 		return (
 			<div className={ sectionClass }>
 				<DocumentHead />
+				<QuerySites allSites />
```

Comment:
Instead of `Layout`, should this maybe live in `my-sites/sidebar` (and use `selectedSiteId` in single-site mode there)? That'd seem a bit more fine-grained to me. Not all sections need (all) sites data...